CMS-Enterprise/mint-app
# [MINT-3187] Add Tdl Status To Echimp

# MINT-3187

## Description


* Add `status` to `GetEchimp` query
* Updated tests
* Update components 

## How to test this change

1.  Navigate to `Plan With CRs and TDLs 1` model collaboration area
2. Verify that the TDL card has status
3. Click `View more` and verify that the sidepanel has the status

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
